bower.json

```diff
@@ -1,36 +1,36 @@
 {
-    "name": "trysmalltalkdemo",
-    "description": "A demo of the trysmalltalk",
-    "version": "0.1.0",
-    "ignore": [
-        "**/.*",
-        "node_modules",
-        "bower_components",
-        "/test_runner.js",
-        "test",
-        "tests"
-    ],
-    "authors": [
-        {
-            "name": "Thomas Rake",
-            "email": "tom.rake@gmail.com"
-        }
-    ],
-    "homepage": "https://github.com/tomrake/trysmalltalk",
-    "keywords": [
-        "Amber",
-        "Smalltalk"
-    ],
-    "license": [
-        "MIT"
-    ],
-    "private": false,
-    "dependencies": {
-        "amber": "^0.13.3",
-        "trysmalltalk": "git://github.com/tomrake/trysmalltalk#0.0.2"
-    },
-    "devDependencies": {
-        "amber-ide-starter-dialog": "^0.1.0",
-        "helios": "^0.3.0"
+  "name": "trysmalltalkdemo",
+  "description": "A demo of the trysmalltalk",
+  "version": "0.1.0",
+  "ignore": [
+    "**/.*",
+    "node_modules",
+    "bower_components",
+    "/test_runner.js",
+    "test",
+    "tests"
+  ],
+  "authors": [
+    {
+      "name": "Thomas Rake",
+      "email": "tom.rake@gmail.com"
     }
-}
\ No newline at end of file
+  ],
+  "homepage": "https://github.com/amber-smalltalk/trysmalltalk",
+  "keywords": [
+    "Amber",
+    "Smalltalk"
+  ],
+  "license": [
+    "MIT"
+  ],
+  "private": false,
+  "dependencies": {
+    "amber": "0.14.3",
+    "trysmalltalk": "0.1.1"
```

Comment:
IOW, do not update bower.json by hand, use `bower install` do it for you. Had you do the simple `bower install trysmalltalk --save`, bower would include other versions as well. IMO it is a bad habit to get used to edit bower.json and then running `bower install`.